lib/v2/douban/other/discussion.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * @Author: nightmare-mio wanglongwei2009@qq.com
+ * @Date: 2023-11-20 23:36:12
+ * @LastEditTime: 2023-11-21 22:13:01
+ * @Description:
+ */
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const link = 'https://book.douban.com/subject';
+    const { data: response } = await got({
+        method: 'get',
+        url: `${link}/${id}/discussion`,
```

Comment:
Use the redirected url 
https://github.com/DIYgod/RSSHub/actions/runs/6945849477/job/18896148171#step:12:17
```suggestion
        url: `${link}/${id}/discussion/`,
```